docs/release_notes/index.md

```diff
@@ -19,11 +19,13 @@ is available in the [commit logs](https://github.com/YosefLab/scvi-tools/commits
     {class}`scvi.train._callbacks.MetricsCallback` {pr}`2136`.
 -   Expose `accelerator` and `device` arguments in {meth}`scvi.hub.HubModel.load_model`
     `pr`{2166}.
+-   Added per-group LFC information in the differential_expression method of the {class}`scvi.model.base.PosteriorPredictiveCheck` {pr}`2173`. The summary dataframe can now be accessed at `metrics["diff_exp"]["summary"]` and the per-group LFC at `metrics["diff_exp"]["lfc_per_model_per_group"]`.
 
 #### Changed
 
 -   Replace `sparse` with `sparse_format` argument in {meth}`scvi.data.synthetic_iid`
     for increased flexibility over dataset format {pr}`2163`.
+-   Added per-group LFC information in the differential_expression method of the {class}`scvi.model.base.PosteriorPredictiveCheck` {pr}`2173`. The summary dataframe can now be accessed at `metrics["diff_exp"]["summary"]` and the per-group LFC at `metrics["diff_exp"]["lfc_per_model_per_group"]`.
```

Comment:
Can probably remove this entry and the one added to 1.0.3

Reply:
I wanted to keep it under "Changed" too, since the type of output object changed (from dataframe to dict). I updated the wording to be a bit more clear.

Reply:
Sounds good. Could you remove the entry under 1.0.3 though since we'll release this with 1.1?